### PR TITLE
feat - added product listing page

### PR DESCRIPTION
**Pull request by akash-1318**
Screenshot 2022-02-12 at 12 31 14 AM
Added the Product listing page using Grid and card component from component library.

### CODE REVIEW

File: pages/styles/products.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
+.product__main-container{
+    display: flex;
+}
```
Comment (haarsh24) [inline pages/styles/products.css:3]: you can declare this property in utli.css , and use it multiple times in class

File: pages/styles/products.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    text-underline-position: under;
+}
+
+.slider {
+    -webkit-appearance: none;
+    width: 100%;
+    height: 10px;
+    border-radius: 5px;
+    background: #d3d3d3;
+    outline: none;
```
Comment (haarsh24) [inline pages/styles/products.css]: Declare this color value and use .

File: pages/styles/products.css [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    margin-top: 1rem;
+  }
+
+  .slider::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    appearance: none;
+    width: 1.5rem;
+    height: 1.5rem;
+    border-radius: 50%;
+    background: #636363;
```
Comment (haarsh24) [inline pages/styles/products.css]: same here

**Review by haarsh24 — APPROVED** [review]
Declare the color and use it in CSS .
Rest all look good , Great and Clean UI .

**Review by yogesh- — APPROVED** [review]
Good to merge

File: pages/styles/products.css (hunk 3 above)
Comment (akash-1318) [reply pages/styles/products.css]: Done with this

**Pull request merged**